triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerLogContext.scala

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
This file provides a trigger focused abstraction layer over `LoggingContextOf[Trigger]`. This allows trigger logging to correctly maintain spans with child and next spans possible.